languages/js/src/Polar.test.ts

```diff
@@ -424,6 +424,16 @@ describe('#loadFiles', () => {
     );
   });
 
+  // test_load_multiple_files_same_name_different_path
+  test('can load two files with the same name but different paths', async () => {
+    const p = new Polar();
+    const filename1 = await tempFile('f();', 'a.polar');
+    const filename2 = await tempFile('g();', 'other/a.polar');
+    await expect(p.loadFiles([filename1, filename2])).resolves.not.toThrow();
+    expect(await query(p, 'f()')).toStrictEqual([map()]);
+    expect(await query(p, 'g()')).toStrictEqual([map()]);
```

Comment:
In JS we create temporary fixtures via the `tempFile()` helper function; in every other language we have persistent fixtures. There's no reason the contents of the tempfiles & the persistent files can't be the same, but the contents is irrelevant — we're just testing that it can successfully load two files with the same name but different paths (& different contents, lest we run afoul of that check).

Reply:
9c84074